Geometry/VeryForwardData/data/CTPPS_2021/Cuts_Per_Region/Reco/v1/RP_Cuts_Per_Region.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+<DDDefinition xmlns="http://www.cern.ch/cms/DDL"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../DDLSchema/DDLSchema.xsd">
+  <SpecParSection label="RP_Cuts_Per_Region.xml" eval="true">
+    <SpecPar name="RP_Silicon_Detectors_Range_Cuts">
+      <PartSelector path="//RP_Silicon_Detector"/>
+      <Parameter name="CMSCutsRegion" value="RPSiliconDetectorsCuts" eval="false"/>
+      <!--Parameter name="ProdCutsForElectrons" value="0.10*mm"/>
+      <Parameter name="ProdCutsForPositrons" value="0.10*mm"/>
+      <Parameter name="ProdCutsForGamma" value="0.10*mm"/-->
```

Comment:
@fabferro , we have here duplicated lines for cuts.

Reply:
The lines are commented, but I can remove them anyway.